src/gol.test.js

```diff
@@ -60,3 +26,228 @@ describe('countLiveNeighbours', () => {
     expect(countLiveNeighbours(input, 3, 2)).toBe(expectedOutput);
   });
 });
+
+const getNeighbours = (grid, i, j) => {
+  const neighbours = [];
+
+  for (let x = -1; x < 2; x += 1) {
+    for (let y = -1; y < 2; y += 1) {
+      if (grid[i + x] !== undefined && grid[i + x][j + y] !== undefined && (x !== 0 || y !== 0)) {
+        neighbours.push([[i + x], [j + y]]);
+      }
+    }
+  }
+
+  return neighbours;
+};
+
+const generateLiveNeighbours = (grid, i, j, liveNeighbours) => {
+  const newGrid = grid;
+  const neighbours = getNeighbours(grid, i, j);
+  if (liveNeighbours > neighbours.length) {
+    return null;
+  }
+  for (let x = 0; x < liveNeighbours; x += 1) {
+    newGrid[neighbours[x][0]][neighbours[x][1]] = 1;
+  }
+
+  return newGrid;
+};
+
+const generateTestData = (size, i, j, liveNeighbours, status) => {
+  const grid = Array(size).fill(0).map(() => Array(size).fill(0));
```

Comment:
Good usage of .fill and .map functions (y). Maybe it will be simpler if you will use Array.from